src/styled-components/SignInContainer.styled.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const SignInContainer = styled.div`
+	position: absolute;
+	top: 0;
+	right: 0;
+	left: 0;
```

Comment:
you are using rigth and left here. also a fixed width, all in all this is too much. usually we are using left or right, top or bottom, width and height are doing the rest for you. Again, it looks not very responsive so far ;)